shared/interfaces/Like.ts

```diff
@@ -0,0 +1,17 @@
+import { IUser } from './User';
+
+export interface ILike {
+  _id: string;
+  author: IUser;
+  dateCreated: Date;
+}
+
+export interface PostLikeCreateData {
```

Comment:
Better use DTO instead of <some-operation>Data naming:
https://www.baeldung.com/java-dto-pattern